packages/pigeon/lib/dart_generator.dart

```diff
@@ -434,6 +452,194 @@ final BinaryMessenger? ${_varNamePrefix}binaryMessenger;
     });
   }
 
+  @override
+  void writeInstanceManager(
+    DartOptions generatorOptions,
+    Root root,
+    Indent indent, {
+    required String dartPackageName,
+  }) {
+    indent.writeln(proxyApiBaseClass);
+
+    indent.writeln(
```

Comment:
Should these calls to templates be `writeln`, or just `write`? It seems odd to use `writeln` for arbitrary multi-line chunks of text.

Reply:
I changed it to `indent.format`